pkg/cmd/repo/create/http.go

```diff
@@ -75,7 +85,8 @@ func repoCreate(client *http.Client, hostname string, input repoCreateInput, tem
 			"input": templateInput,
 		}
 
-		err := apiClient.GraphQL(hostname, `
+		err := apiClient.GraphQL(
```

Comment:
Tip: please do not include changes that merely change the style of existing code. Use `go fmt ./...` before submitting your PRs in the future.

Reply:
Sory